src/components/SignupForm/SignupForm.jsx

```diff
@@ -71,38 +142,140 @@ export default function SignupForm({ setTypeUser, typeUser }) {
+                        placeholder="Email"
+                        size="small"
+                        style={{ width: 280 }}
+                        required
+                        autoComplete="email"
+                        autoFocus
+                        value={email}
+                        error={!!emailErrorText}
+                        helperText={emailErrorText}
+                        onChange={(e) => setEmail(e.target.value)}
+                        margin="dense"
+                        type="text"
+                        InputProps={{
+                            startAdornment: (
+                                <InputAdornment position="start">
+                                    <PersonIcon sx={iconStyle} />
+                                </InputAdornment>
+                            ),
+                        }}
+                    />
+                    <styleMui.Input
+                        placeholder="Họ"
+                        size="small"
+                        style={{ width: 280 }}
+                        required
+                        autoComplete="lastName"
+                        autoFocus
+                        value={lastName}
+                        error={!!lastNameErrorText}
+                        helperText={lastNameErrorText}
+                        onChange={(e) => setLastName(e.target.value)}
+                        margin="dense"
+                        type="text"
+                        InputProps={{
+                            startAdornment: (
+                                <InputAdornment position="start">
+                                    <BadgeIcon sx={iconStyle} />
+                                </InputAdornment>
+                            ),
+                        }}
+                    />
                     <styleMui.Input
-                        id="margin-dense"
-                        placeholder={item.text}
-                        key={indx}
+                        placeholder="Tên"
                         size="small"
-                        fullWidth
                         style={{ width: 280 }}
+                        required
+                        autoFocus
+                        value={firstName}
+                        error={!!firstNameErrorText}
+                        helperText={firstNameErrorText}
+                        onChange={(e) => setFirstName(e.target.value)}
                         margin="dense"
-                        variant="outlined"
+                        type="text"
                         InputProps={{
                             startAdornment: (
                                 <InputAdornment position="start">
-                                    {item.icon}
+                                    <BadgeIcon sx={iconStyle} />
                                 </InputAdornment>
                             ),
                         }}
                     />
```

Comment:
Look like we have same tag here, so you need put this into array and map, that will be more cool.